entrypoint.sh

```diff
@@ -17,9 +17,7 @@ then
 elif [ "$1" == "api" ]
 then
   shift 1
-  # TODO: This is fine for local development but this file will soon need to change to have
-  # gunicorn infront of flask
-  flask run --host 0.0.0.0 --port 6011
+  gunicorn -c /home/vcap/app/gunicorn_config.py application
```

Comment:
So this is good but as you pointed out in PR message, this will change `make run-flask-with-docker` to actually run gunicorn infront of flask.

I think we should firstly change the name of the command so that it is `make run-gunicorn-with-docker` so it says what it does. 

And then secondly, I'm wondering if we should then add back in a command to `make run-flask-with-docker` so we can still have a way for just running flask...

Reply:
I think we should to avoid breaking devs workflows. As Leo [mentioned](https://github.com/alphagov/notifications-api/pull/3726#discussion_r1118666841) flask also runs with debug mode enabled